pkg/virt-api/webhooks/utils.go

```diff
@@ -74,6 +75,18 @@ type Informers struct {
 	NamespaceInformer  cache.SharedIndexInformer
 }
 
-func IsARM64(vmiSpec *v1.VirtualMachineInstanceSpec) bool {
-	return vmiSpec.Architecture == "arm64"
+func IsDefaultArch(vmiSpec *v1.VirtualMachineInstanceSpec, arch string, config *virtconfig.ClusterConfig) bool {
+	return vmiSpec.Architecture == arch || (vmiSpec.Architecture == "" && config.GetConfig().ArchitectureConfiguration.DefaultArchitecture == arch)
+}
+
+func IsAMD64(vmiSpec *v1.VirtualMachineInstanceSpec, config *virtconfig.ClusterConfig) bool {
+	return IsDefaultArch(vmiSpec, "amd64", config)
```

Comment:
```suggestion
	return vmiSpec.Architecture == "amd64" || IsDefaultArch(vmiSpec, "amd64", config)
```
Of course the same also for the other arches.